src/modules/spaces/queries/find-space-by-cp-admin-id.graphql

```diff
@@ -0,0 +1,41 @@
+query findSpaceByCpAdminId($cpAdminId: uuid!) {
```

Comment:
This isn't correct
A maintainer will have its own slug as a separate column. the schema_identifier is the organisation id, we used this id as a temporary placeholder
To fix this you'll have to add a slug to the space table and filter on that value instead.